src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfiguration.java

```diff
@@ -180,19 +180,19 @@ private static FormValidation checkAdminCredentialsId(String adminCredentialsId)
      */
     private static FormValidation checkBaseUrl(String baseUrl) {
         if (isEmpty(baseUrl)) {
-            return FormValidation.error("Enter your Bitbucket instance's URL.");
+            return FormValidation.error("Required");
```

Comment:
Hi @dkjellin . I had something like this previously, but I decided to make things simple and just make it 'required' as otherwise just for this field the user is shown: 
1. the label - Instance name
2. help text - To help users identify this instance 
3. validation - Enter your bitbucket instance 

I'd prefer to leave this as is if that's ok with you.

Reply:
Sure, like I said it felt very short. I looked at a few others and they are all a bit longer the git plugin has `Please enter Git repository` for example. 
I'm ok with leaving it, but my preference would be to change it, but it is your call.

Reply:
A small technical consideration we may want to consider is the `FormValidation` message can be used internally (for example when we verify if a given server configuration is correct). Right now it doesn't matter and YAGNI but worth considering if we're, for example, diagnosing why a server config is invalid from within the plugin.

Reply:
@dkjellin due to how noisy the UI is I'd prefer to leave it as is for now, and to see how people react at the blitz. Thanks for your help.